pkg/archive/parser_test.go

```diff
@@ -110,11 +111,161 @@ func TestParse(t *testing.T) {
 
 			res, err := p.Parse(tc.dir)
 			if tc.wantErr != nil {
-				assert.Equal(t, err.Error(), tc.wantErr.Error())
+				assert.Equal(t, tc.wantErr.Error(), err.Error())
 			} else {
 				assert.Nil(t, err)
 				assert.Equal(t, tc.want, res)
 			}
 		})
 	}
 }
+
+func TestParseGroupVersions(t *testing.T) {
+	tests := []struct {
+		name    string
+		files   []string
+		buDir   string
```

Comment:
What does `bu` stand for here?

Reply:
It stands for `backup`. I'll change that to `backupDir` because `buDir` is too unclear.